src/ex2/style.css

```diff
@@ -1,117 +1,207 @@
-  border: 1px silver solid;
-  border-radius: 8px;
-  -webkit-box-shadow: 12px 12px 38px -13px #000000;
-  box-shadow: 12px 12px 38px -13px #000000;
+  height: 100vh;
+  background-image: linear-gradient(120deg, lightblue, rgb(45, 111, 133));
 }
+  input[type=text] {
+    padding-left: 15px;
+    width: 79%;
+    padding: 15px;
+    margin: 5px 0 22px 0;
+    border: none;
+    background: #f1f1f1;
+  }
+  
+  input[type=text]:focus {
+    background-color: #ddd;
+    outline: none;
+  }
 
-.list-controls {
-  padding: 0px 32px;
-  background-color: white;
-  display: flex;
-  align-items: center;
-  border-top-right-radius: 8px;
-  border-top-left-radius: 8px;
+#footer {
+  display: inline; 
+  width: 80px;
+  height: 90px;
 }
+#footer #h3 {
+    font-size: 4px;
+     margin :30px;
+  }
+#buttonClearAll {
+    background-color: #0095ff;
+    border: 1px solid transparent;
+    border-radius: 3px;
+    box-shadow: rgba(255, 255, 255, .4) 0 1px 0 0 inset;
+    box-sizing: border-box;
+    color: #fff;
+    cursor: pointer;
+    font-size: 10px;
+    padding: 6px 18px;
+    pointer-events: none;
+    opacity: 0;
+  }  
+#buttonClearAll.hover,
+#buttonClearAll.focus {
+    background-color: #07c;
+    pointer-events: auto;
 
-.list-controls #list-item-input {
-  width: 85%;
-  height: 30px;
-  font-size: 16px;
-  padding-left: 8px;
-}
-
-.list-controls #list-item-submit {
-  color: white;
-  background-color: #6c6cff;
-  border-style: solid;
-  border-color: transparent;
-  border-radius: 4px;
-  height: 35px;
-  width: 35px;
-  font-size: 32px;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  cursor: pointer;
-  margin-left: auto;
-}
+  }
+  
+#buttonClearAll.focus {
+    box-shadow: 0 0 0 4px rgba(0, 149, 255, .15);
+    pointer-events: auto;
+  }
+  
+#buttonClearAll.active {
+    box-shadow: none;
+    opacity: 1;
+    pointer-events: auto;
+    align-items: right;
+    margin :10px;
+  }
+     
+  .card {
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+    transition: 0.3s;
+    width: 85%;
+    margin: auto;
+    width: 30%;
+    border: 3px solid rgb(0, 62, 128);
+    padding: 10px;
+    background-color: rgb(255, 255, 255);
+    align-items: center;
+    overflow: clip;
+  
+  }
 
-.list-controls #list-item-submit:hover {
-  background-color: #3c3cff;
-  transition: background-color 0.2s ease-in-out;
+  .card:hover {
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+  }
+  .card #buttonAdd {
+    background-color: #0095ff;
+    border: 1px solid transparent;
+    border-radius: 3px;
+    box-shadow: rgba(255, 255, 255, .4) 0 1px 0 0 inset;
+    box-sizing: border-box;
+    color: #fff;
+    cursor: pointer;
+    font-size: 19px;
+    font-weight: 400;
+    line-height: 1.15385;
+    margin: 0;
+    outline: none;
+    padding: 10px;
+    position: relative;
+    text-align: center;
+    text-decoration: none;
+    user-select: none;
+    -webkit-user-select: none;
+    touch-action: manipulation;
```

Comment:
it's good to take references for CSS from other places - just make sure you copy only needed code. This file contains a lot CSS and probably a few unintentional behaviors